exp/api/v1beta1/doc.go

```diff
@@ -14,4 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package v1beta1 contains experimental v1beta1 API implementation.
+// +k8s:openapi-gen=true
```

Comment:
```suggestion
```
Let's not use openapi gen here. We only did this in the clusterv1 package for RuntimeSDK, because some hooks are using clusterv1.Cluster (just fyi, you didn't get a diff as it's not configured for this pkg (Makefile > generate-go-openapi))

Reply:
ah, sorry. Was just copy pasting the comment and though this was required for something.